pkg/eventmanager/eventmanager.go

```diff
@@ -183,6 +183,8 @@ func (s *eventManager) NotifyResult(state notifier.MuoState, result string) erro
 	switch state {
 	case notifier.MuoStateHealthCheckSL:
 		description = fmt.Sprintf(UPGRADE_HEALTHCHECK_DELAY_DESC, uc.Spec.Desired.Version, result)
+	case notifier.MuoStatePreHealthCheckSL:
+		description = fmt.Sprintf(UPGRADE_HEALTHCHECK_DELAY_DESC, uc.Spec.Desired.Version, result)
```

Comment:
I think we should update the description as well. The upgrade getting delayed for "Upgrading" phase makes sense but for this PHC in "New" state, there's no delay since the upgrade hasn't even started at all. We should have a description along the lines of "New cluster upgrade is scheduled for more than 2 hours and healthcheck status is failing for now. The failures need to be fixed before upgrade begins to have seamless upgrade experience".

Reply:
Agree